### PR TITLE
Firestore Emulator Release v1.3.0

### PR DESCRIPTION
**Pull request by scottcrossen**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -3,9 +3,11 @@ fixed - Fixed Firestore emulator bug regarding array ordering during writing/rea
 fixed - Fixed Firestore emulator handling of query cursors using document names
 fixed - Fixed a race condition when deploying Firestore indexes (issues #1080 and #1081)
 fixed - Fixed an error that occurs when a Firestore field override removes all indexes
+fixed - Fixed Firestore service routine to allow readTime in 'getDocument'
```
Comment (mbleigh) [inline changelog.txt]: should the word "emulator" be in this line somewhere?

File: changelog.txt [hunk 2]
```diff
@@ -3,9 +3,11 @@ fixed - Fixed Firestore emulator bug regarding array ordering during writing/rea
...
 feature - Firestore emulator now has the ability to produce rule-coverage reports
+feature - Firestore emulator now has 'clearDatabase' endpoint
```
Comment (mbleigh) [inline changelog.txt]: Shouldn't the emulator version be `1.3.0` if there's a new feature?

**Comment by scottcrossen** [comment]
I'd like Ryan's approval before merge

**Comment by scottcrossen** [comment]
I updated the release notes based on Ryan's comments in another code-review.
Also, yeah I won't forget to squash and merge. I forgot once in a different repository and will never forget again (it's also the default here)

**Pull request merged**